src/main/java/com/extendedclip/papi/expansion/mcstatistics/StatisticsExpansion.java

```diff
@@ -20,233 +20,277 @@
+            }
+
+            case "use_item": {
+                return calculateTotal(player, Statistic.USE_ITEM);
+            }
+
+            case "break_item": {
+                return calculateTotal(player, Statistic.BREAK_ITEM);
+            }
+
+            case "craft_item": {
+                return calculateTotal(player, Statistic.CRAFT_ITEM);
+            }
+
+            /*
+             * Time played
+             */
+            case "time_played": {
+                return StatisticsUtils.formatTime(Duration.of(secondsPlayed, ChronoUnit.SECONDS));
+            }
+
+            case "time_played:seconds": {
+                //return seconds != 0 ? Long.toString(TimeUnit.MINUTES.toSeconds(1) - seconds) : "0";
+                return Integer.toString(secondsPlayed % 60);
+            }
+
+            case "time_played:minutes": {
+                //return minutes != 0 ? Long.toString(TimeUnit.HOURS.toMinutes(1) - minutes) : "0";
+                return Long.toString(TimeUnit.SECONDS.toMinutes(secondsPlayed) % 60);
+            }
+
+            case "time_played:hours": {
+                //return hours != 0 ? Long.toString(TimeUnit.DAYS.toHours(1) - hours) : "0";
+                return Long.toString(TimeUnit.SECONDS.toHours(secondsPlayed) % 24);
+            }
+
+            case "seconds_played": {
+                return Integer.toString(secondsPlayed);
+            }
+
+            case "minutes_played": {
+                return Long.toString(TimeUnit.SECONDS.toMinutes(secondsPlayed));
+            }
+
+            case "hours_played": {
+                return Long.toString(TimeUnit.SECONDS.toHours(secondsPlayed));
+            }
+
+            case "time_played:days":
+            case "days_played": {
+                return Long.toString(TimeUnit.SECONDS.toDays(secondsPlayed));
+            }
+
+            /*
+             * Time since last death
+             */
+            case "time_since_death": {
+                return StatisticsUtils.formatTime(Duration.of(secondsSinceLastDeath, ChronoUnit.SECONDS));
+            }
+
+            case "seconds_since_death": {
+                return Integer.toString(secondsSinceLastDeath);
+            }
+
+            case "minutes_since_death": {
+                return Long.toString(TimeUnit.SECONDS.toMinutes(secondsSinceLastDeath));
+            }
+
+            case "hours_since_death": {
+                return Long.toString(TimeUnit.SECONDS.toHours(secondsSinceLastDeath));
+            }
+
+            case "days_since_death": {
+                return Long.toString(TimeUnit.SECONDS.toDays(secondsSinceLastDeath));
+            }
         }
-      case "minutes_played":
-        try {
-          return String.valueOf((p.getStatistic(Statistic.PLAY_ONE_MINUTE) / 20) / 60);
-        } catch (NoSuchFieldError error) {
-          return String.valueOf((p.getStatistic(Statistic.valueOf("PLAY_ONE_TICK")) / 20) / 60);
+
+        final int splitterIndex = identifier.indexOf(":");

```

Comment:
Can be `identifier.indexOf(':');` since it's a single character instead of a full string.